packages/devtools_app/test/cpu_profiler_controller_test.dart

```diff
@@ -16,7 +16,7 @@ import 'test_data/cpu_profile_test_data.dart';
 
 void main() {
   group('CpuProfileController', () {
-    CpuProfilerController controller;
+    late CpuProfilerController controller;
     FakeServiceManager fakeServiceManager;
```

Comment:
late

Reply:
done

Reply:
why doesn't `fakeServiceManager` have to be late here? I'm surprised this isn't giving an analyzer warning